src/SprykerEco/Zed/Payone/Business/Payment/MethodSender/PayoneGenericRequestMethodSender.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Payone\Business\Payment\MethodSender;
+
+use Generated\Shared\Transfer\PayonePaypalExpressCheckoutGenericPaymentResponseTransfer;
+use Generated\Shared\Transfer\PayoneStandardParameterTransfer;
+use SprykerEco\Zed\Payone\Business\Api\Adapter\AdapterInterface;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\GenericPaymentContainer;
+use SprykerEco\Zed\Payone\Business\Api\Response\Container\GenericPaymentResponseContainer;
+use SprykerEco\Zed\Payone\Business\Payment\DataMapper\StandartParameterMapperInterface;
+
+class PayoneGenericRequestMethodSender implements PayoneGenericRequestMethodSenderInterface
+{
+    /**
+     * @var \SprykerEco\Zed\Payone\Business\Api\Adapter\AdapterInterface
+     */
+    protected $executionAdapter;
+
+    /**
+     * @var \Generated\Shared\Transfer\PayoneStandardParameterTransfer
+     */
+    protected $standardParameter;
+
+    /**
+     * @var \SprykerEco\Zed\Payone\Business\Payment\DataMapper\StandartParameterMapperInterface
+     */
+    protected $standartParameterMapper;
+
+    /**
+     * @param \SprykerEco\Zed\Payone\Business\Api\Adapter\AdapterInterface $executionAdapter
+     * @param \Generated\Shared\Transfer\PayoneStandardParameterTransfer $standardParameter
+     * @param \SprykerEco\Zed\Payone\Business\Payment\DataMapper\StandartParameterMapperInterface $standartParameterMapper
+     */
+    public function __construct(
+        AdapterInterface $executionAdapter,
+        PayoneStandardParameterTransfer $standardParameter,
+        StandartParameterMapperInterface $standartParameterMapper
+    ) {
+        $this->executionAdapter = $executionAdapter;
+        $this->standardParameter = $standardParameter;
+        $this->standartParameterMapper = $standartParameterMapper;
+    }
+
+    /**
+     * @param \SprykerEco\Zed\Payone\Business\Api\Request\Container\GenericPaymentContainer $requestContainer
+     *
+     * @return \Generated\Shared\Transfer\PayonePaypalExpressCheckoutGenericPaymentResponseTransfer
+     */
+    public function performGenericRequest(GenericPaymentContainer $requestContainer): PayonePaypalExpressCheckoutGenericPaymentResponseTransfer
+    {
+        $this->standartParameterMapper->setStandardParameter($requestContainer, $this->standardParameter);
+
+        $rawResponse = $this->executionAdapter->sendRequest($requestContainer);
+        $responseContainer = new GenericPaymentResponseContainer($rawResponse);
+        $responseTransfer = new PayonePaypalExpressCheckoutGenericPaymentResponseTransfer();
```

Comment:
* Extract to a method `mapToPayonePaypalExpressCheckoutGenericPaymentResponseTransfer().

Reply:
++